tests/infra/node.py

```diff
@@ -188,38 +188,38 @@ def get_sealed_secrets(self):
         return self.remote.get_sealed_secrets()
 
     def user_client(self, format="msgpack", user_id=1, **kwargs):
-        return infra.jsonrpc.client(
+        return infra.client.client(
```

Comment:
Nitpick: `infra.client.client` is a little awkward - `infra.clients`?

Reply:
Or perhaps the function should be `make_client`

Reply:
I went for `infra.clients` as it supports different clients for now (requests, curl and framed TLS).